plugins/modules/system/listen_ports_facts.py

```diff
@@ -72,11 +84,21 @@
       returned: if TCP servers were found
       type: list
       contains:
-        address:
+        local_address:
           description: The address the server is listening on.
           returned: always
           type: str
           sample: "0.0.0.0"
+        foreign_address:
+          description: The address of the remote end of the socket.
+          returned: if ``include_non_listening`` is set
```

Comment:
```suggestion
          returned: if I(include_non_listening) is set
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation